presto-hive/src/test/java/com/facebook/presto/hive/TestHiveClientConfig.java

```diff
@@ -78,6 +78,7 @@ public void testDefaults()
                 .setMaxPartitionsPerWriter(100)
                 .setMaxOpenSortFiles(50)
                 .setWriteValidationThreads(16)
+                .setMaxLineLength(new DataSize(100, DataSize.Unit.MEGABYTE))
```

Comment:
The `DataSize` prefix is not necessary since `Unit` is already imported. Your IDE should have a warning for this.